health/micro-ui-react19/web/packages/css/src/pages/employee/commodityManagement.scss

```
@@ -0,0 +1,491 @@
+
+.cm-summary-title-block {
+  max-width: 40%;
+}
+
+.cm-summary-title {
+  color: var(--digitv2-lightTheme-text-primary);
+  font-family: Roboto;
+  font-weight: 700;
+  font-size: 28px;
+  letter-spacing: 0px;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)